dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/persistence/tlvformat/TLVBlockFileReaderTest.kt

```diff
@@ -88,6 +89,7 @@ internal class TLVBlockFileReaderTest {
     @Test
     fun `M log error W read() { invalid TLV type }`() {
         // Given
+        assumeTrue(fakeDataString.length > 2)
```

Comment:
Minor:  this assumption can fix flaky test, but it seems this case worth being covered as well.  we can have a if else condition to test if the length is smaller or equal than 2, `internalLogger` should output error which starts with "Failed to deserialize TLV data length".